Raven.Voron/Voron/Trees/Page.cs

```diff
@@ -172,6 +172,19 @@ public ushort PageSize
             }
         }
 
+        public List<long> GetAllOverflowPages()
+        {
+            var results = new List<long>(NumberOfEntries);
```

Comment:
Note that if you have the NumberOfEntries, you already have paged the page into memory